docs/proposals/20181121-machine-api.md

```diff
@@ -42,9 +42,9 @@ performed in-place or via Node replacement.
 ## Proposal
 
 This proposal introduces a new API type: Machine. See the full definition in
-[types.go](types.go).
+[machine_types.go](https://github.com/kubernetes-sigs/cluster-api/tree/release-0.2/pkg/apis/deprecated/v1alpha1/machine_types.go).
```

Comment:
Maybe we shouldn't link to release-0.2

Reply:
Thanks, the proposal is old, should we use the old API or master branch?

Reply:
The hyperlinks should also be modified in
https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/proposals/20181121-machine-api.md#types

Reply:
Let's wait for feedback from other folks. I'm not sure what the policy is.